packages/packages/top-bar/src/index.ts

```diff
@@ -1,13 +1,22 @@
 export {
+	getMenuId,
 	registerLink,
 	registerAction,
 	registerToggleAction,
 	injectIntoCanvasDisplay,
 	injectIntoPrimaryAction,
 } from './locations/index';
 
+export type { MenuName, MenuGroup, Menus } from './locations/menus';
+
 export * from './types';
 
+export { default as PopoverMenu } from './components/ui/popover-menu';
+export { default as PopoverMenuItem } from './components/ui/popover-menu-item';
+export { default as ToolbarMenu } from './components/ui/toolbar-menu';
+export { default as ToolbarMenuItem } from './components/ui/toolbar-menu-item';
+export { default as ToolbarMenuToggleItem } from './components/ui/toolbar-menu-toggle-item';
+
```

Comment:
Let's be sure with @eyalnathan and @rotemee that we want to expose all of those things and make them public, and we will need to support this API.